Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleViewportEdit.cpp

```diff
@@ -119,9 +119,11 @@ namespace AzToolsFramework
         {
             const float radius = GetCapsuleRadius();
             const AZ::Transform localTransform = GetLocalTransform();
-            const AZ::Transform manipulatorSpace = GetManipulatorSpace();
+            const AZ::Transform localInverse = localTransform.GetInverse();
+            const AZ::Transform manipulatorSpaceInverse = GetManipulatorSpace().GetInverse();
+
             const AZ::Vector3 inverseTransformedForward =
-                (manipulatorSpace * localTransform).GetInverse().TransformVector(cameraState.m_forward);
+                localInverse.TransformVector(manipulatorSpaceInverse.TransformVector(cameraState.m_forward) / GetNonUniformScale());
```

Comment:
nit: assert to prevent divide from zero
```suggestion
                auto nonUniformScale = GetNonUniformScale();
                AZ_Assert(nonUniformScale != 0, "nonUniformScale must be non-zero");
                localInverse.TransformVector(manipulatorSpaceInverse.TransformVector(cameraState.m_forward) / nonUniformScale);
```

Reply:
There are a lot of guards to prevent scale from being zero.

Reply:
I've added another guard now in GetNonUniformScale.